containers/README.md

```
@@ -156,23 +156,23 @@ docker build -f containers/Dockerfile.cuda \
 ```bash
 # Run CI unit tests in a container
-make test-ci-container
+mise run test:ci-container
 
 # Verify mise-managed tools install correctly (fast -- setup stage only)
-make test-tool-install
+mise run test:tool-install
 ```
 
-### Makefile Targets
+### Mise Tasks
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)